src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py

```diff
@@ -396,20 +396,6 @@ def ensure_container_insights_for_monitoring(
                     error = e
             else:
                 raise error
```

Comment:
we should remove below code since this  is not required since you have removed DCR and DCR-A region check
  # check if region supports DCRs and DCR-A
            for _ in range(3):
                try:
                    feature_check_url = cmd.cli_ctx.cloud.endpoints.resource_manager + \
                        f"/subscriptions/{cluster_subscription}/providers/Microsoft.Insights?api-version=2020-10-01"
                    r = send_raw_request(cmd.cli_ctx, "GET", feature_check_url)
                    error = None
                    break
                except AzCLIError as e:
                    error = e
            else:
                raise error

https://github.com/Azure/azure-cli/blob/7c00275bc73d4bc4794f79a78caaf4678340bcae/src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py#L387C12-L398C28